lms/static/styles/mixins/focus.scss

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
This file was copied from a file of the same name in the client repo. In future I think we'll want to extract this into a shared package.